meta-facebook/yv35-ji/src/platform/plat_pldm_fw_update.c

```diff
@@ -406,6 +438,16 @@ static bool get_vr_fw_version(void *info_p, uint8_t *buf, uint8_t *len)
 		goto post_hook_and_ret;
 	}
 
+	/* Get vr remain wr count if using non-INF chip */
+	if (strncmp(vendor_name_p, "INF ", 4)) {
```

Comment:
Please address this cpp check failure as its added as a side-effect of this change. Thanks

https://github.com/facebook/OpenBIC/actions/runs/9906750548/job/27368970138

meta-facebook/yv35-ji/src/platform/plat_pldm_fw_update.c:442:14:Either the condition '!vendor_name_p' is redundant or there is possible null pointer dereference: vendor_name_p.